webapp/src/Controller/Jury/JudgehostController.php

```diff
@@ -83,6 +83,7 @@ public function indexAction(Request $request): Response
             ->from(Judgehost::class, 'j')
             ->leftJoin('j.restriction', 'r')
             ->select('j', 'r')
+            ->andWhere('j.hidden = 0')
```

Comment:
Not really important but I think you could write false instead of 0 here

Reply:
We use in most places 0/1, don't we?

Reply:
Not sure but if we do keep it indeed